src/views/profile/edit.ejs

```diff
@@ -247,57 +248,58 @@
         <option value="PT">Portugal</option>
         <option value="PR">Puerto Rico</option>
         <option value="QA">Qatar</option>
+        <option value="CG">Republic of the Congo</option>
```

Comment:
Huh, looks like we have 3 Congos?

Reply:
Should remove this one and keep Congo imo. DRC is a different country.

Reply:
Thanks!